updater/spec/dependabot/updater/operations/group_update_all_versions_spec.rb

```diff
@@ -289,4 +289,72 @@
       group_update_all.perform
     end
   end
+
+  context "when the snapshot is updating vendored dependencies", :vcr do
+    let(:job) do
+      Dependabot::Job.new_update_job(
+        job_id: "1558782000",
+        job_definition: job_definition_with_fetched_files,
+        repo_contents_path: create_temporary_content_directory(fixture: "bundler_vendored")
+      )
+    end
+
+    let(:job_definition) do
+      job_definition_fixture("bundler/version_updates/group_update_all_with_vendoring")
+    end
+
+    let(:dependency_files) do
+      # Let's use the already up-to-date files
+      original_bundler_files(fixture: "bundler_vendored")
+    end
+
+    before do
+      stub_rubygems_calls
+    end
+
+    it "creates a pull request that includes changes to the vendored files" do
+      expect(mock_service).to receive(:create_pull_request) do |dependency_change|
+        expect(dependency_change.dependency_group.name).to eql("everything-everywhere-all-at-once")
+
+        # We updated the right dependencies
+        expect(dependency_change.updated_dependencies.length).to eql(2)
+        expect(dependency_change.updated_dependencies.map(&:name)).to eql(%w(dummy-pkg-b dummy-git-dependency))
+
+        # We updated the right files correctly.
+        expect(dependency_change.updated_dependency_files_hash.length).to eql(8)
+
+        # We've updated the gemfiles properly
+        gemfile = dependency_change.updated_dependency_files.find { |file| file.path == "/Gemfile" }
+        expect(gemfile.content).to eql(fixture("bundler_vendored/updated/Gemfile"))
+        gemfile_lock = dependency_change.updated_dependency_files.find { |file| file.path == "/Gemfile.lock" }
+        expect(gemfile_lock.content).to eql(fixture("bundler_vendored/updated/Gemfile.lock"))
+
+        # We've deleted the old version of dummy-pkg-b
+        old_dummy_pkg_b = dependency_change.updated_dependency_files.find do |file|
+          file.path == "/vendor/cache/dummy-pkg-b-1.1.0.gem"
+        end
+        expect(old_dummy_pkg_b.operation).to eql("delete")
+
+        # We've created the new version of dummy-pkg-b
+        new_dummy_pkg_b = dependency_change.updated_dependency_files.find do |file|
+          file.path == "/vendor/cache/dummy-pkg-b-1.2.0.gem"
+        end
+        expect(new_dummy_pkg_b.operation).to eql("create")
+
+        # We've deleted the old version of the vendored git dependency
+        old_git_dependency_files = dependency_change.updated_dependency_files.select do |file|
+          file.path.start_with?("/vendor/cache/ruby-dummy-git-dependency-20151f9b67c8")
+        end
+        expect(old_git_dependency_files.map(&:operation)).to eql(%w(delete delete))
```

Comment:
for other reviewers: the two `delete` operations here correlate to the bundlecache and gemspec files being deleted.